src/core/styles/properties.css

```diff
@@ -2,11 +2,11 @@
   :root {
     /* Neutral colors */
     --color-neutral-000: #ffffff;
-    --color-neutral-100: #f8fafc;
-    --color-neutral-200: #f4f8fb;
-    --color-neutral-300: #edf1f7;
+    --color-neutral-100: #f6f8fa;
+    --color-neutral-200: #eef1f6;
+    --color-neutral-300: #e6eaf0;
```

Comment:
Do we need to check the pages that affect these changes? or not a lot of difference between these colors?

Reply:
Pretty subtle changes - we can just roll this out